stacktest/fakemetrics/out/kafkamdm/kafkamdm.go

```diff
@@ -18,22 +18,23 @@ import (
 
 type KafkaMdm struct {
 	out.OutStats
-	topic      string
-	brokers    []string
-	config     *sarama.Config
-	client     sarama.SyncProducer
-	hash       hash.Hash32
-	part       *p.Kafka
-	lmPart     LastNumPartitioner
-	partScheme string
+	topic         string
+	brokers       []string
+	config        *sarama.Config
+	client        sarama.SyncProducer
+	hash          hash.Hash32
+	part          *p.Kafka
```

Comment:
```suggestion
	part          p.Partitioner
```
if you just use the Partitioner interface, you dont need to also define `lmpart`